blog/src/components/Ratings.js

```diff
@@ -0,0 +1,85 @@
+import React, { useEffect } from 'react';
+// @author Alan Harder
+
+function loaddata() {
+  var script = document.createElement('SCRIPT');
+  script.type = 'text/javascript';
+  script.src = 'https://rating.jenkins.io/rate/result.php';
+  script.onload = function() { do_loaddata(); }
+  script.onreadystatechange = function() { // For IE
+    if (this.readyState == 'loaded' || this.readyState == 'complete') do_loaddata();
+  }
+  document.getElementById('head').appendChild(script);
+  document.getElementById('ratings').style.display = 'block';
+  return false;
+}
+
+function health(nm, cls, ver, rate, desc) {
+  return '<img src="../../images/images/changelog/' + nm + '.svg" onclick="rate(\'' + ver + '\',' + rate + ')" class="rate ' + cls + '" alt="' + nm + '" title="' + desc + '"/>';
+}
+
+function do_loaddata() {
+  var r, v, j = true, div1, div2, txt;
+  for (var anchors = document.getElementsByTagName('H3'), i = 0; i < anchors.length; i++) {
+    if (anchors[i].id.charAt(0) != 'v') continue;
+    r = data[v = anchors[i].id.substring(1)];
```

Comment:
Did this code work for you @Vandit1604? 

I am currently getting the following errors in my stack trace when I tried to run the code locally:
```
/Users/krisstern/jenkins-docs/blog/src/components/Ratings.js
  25:9   error  'data' is not defined         no-undef
  59:50  error  Unexpected use of 'location'  no-restricted-globals
  62:25  error  Unexpected use of 'location'  no-restricted-globals
```

The script is not working for me.

Reply:
Yeah. I'm getting the same errors. I'm working on it right now.

Reply:
I added a commit to patch them for now, but it appears that the three buttons are not working on say the Changelog page. There is no reaction after I clicked on each.